lib/shopify_transporter/pipeline/magento/order/discounts.rb

```diff
@@ -82,9 +82,11 @@ def value_as_float(hash, key)
           def qualifies_for_percentage_discount?(hash)
             return false unless line_items?(hash)
 
-            return true if line_items(hash).is_a?(Hash) && value_as_float(line_items(hash), 'discount_percent') > 0
-
-            all_discount_percentages(hash).length == 1 && all_discount_percentages(hash).first > 0
+            if line_items(hash).is_a?(Hash)
```

Comment:
i don't understand how this changes any of the behaviour

Reply:
We're not calling `value_as_float(line_items(hash), 'discount_percent') > 0`